source/support/rotate-secrets/index.html.md.erb

```diff
@@ -0,0 +1,180 @@
+---
+title: Rotate a secret
+weight: 35
+last_reviewed_on: 2021-01-19
+review_in: 6 months
+---
+
+# Rotate a secret
+
+If a secret expires or is compromised, you will need to rotate it.
+This is a currently a manual process.
+
+This content explains:
+
+- how secrets are used
+- what secrets we have
+- how to rotate secrets if needed
+
+If a secret is compromised, this is a security incident. Follow [the
+incident management process][] as well as rotating the secret.
+
+[the incident management process]: /incident
+
+## Prerequisites
+
+You must [install the gds-cli tool][] to manage secrets.
```

Comment:
Would I be right in thinking you must also not be on probation?

Reply:
Added that you need production access